src/main/scala/com/twitter/finagle/postgres/values/ValueEncoder.scala

```diff
@@ -189,6 +189,17 @@ object ValueEncoder extends LowPriorityEncoder {
     m: Map[String, String] => m.mapValues(Option(_))
   }
 
+  implicit val jsonb: ValueEncoder[Array[Byte]] = instance[Array[Byte]](
```

Comment:
The problem is that there's already an encoder for `Array[Byte]`, for bytea. Putting this here will prevent both of them from working.

Maybe we could use a value class to represent jsonb? `case class JSONB(bytes: Array[Byte]) extends AnyVal` - then `JSONB` could get its own encoder/decoder.

Reply:
Do you also have code to work with JSONB byte arrays? The value class would be a good place for methods that go i.e. `jsonb_bytes <-> json_string`.

Reply:
Added.